NMS-12435: Associate exporter node using Observation Domain Id (#2847)
OpenNMS/opennms: opennms-doc/guide-admin/src/asciidoc/text/flows/setup.adoc

NOTE: The associated protocol, in this case `Netflow-5` must also be enabled on _{opennms-product-name}_ for the messages to be processed.

> In some scenarios the exporters address is altered due to network address translation. In this case you can use node meta-data to identify the exporter.

Reviewer: ```suggestion
In some scenarios the exporters address is altered due to network address translation.
In this case you can use node meta-data to identify the exporter.
```

Author: done

> Use the parameter `metaDataNodeLookup` to specify a context-key pair `context:key` for the lookup.

Reviewer: ```suggestion
Use the `metaDataNodeLookup`  parameter to specify a context-key pair in the form of `context:key` for the lookup.
```

Author: done

> This value is used to match the Id of the various protocols:

Reviewer: ```suggestion
This value used for the lookup corresponds to the following fields from the various protocols:
```

Author: done
